casepropods/family_connect_registration/plugin.py

```diff
@@ -107,8 +107,8 @@ def has_whatsapp_account(self, number):
         res = requests.post(
             self.config.engage_url,
             json={
-                "contacts": [number],
-                "blocking": "wait"
+                "blocking": "wait",
+                "contacts": ["number"]
```

Comment:
You can undo this change, number should not be in quotes.

Reply:
@erikh360 Changed